cypress/e2e/trafficlight/trafficlight.spec.ts

```diff
@@ -70,232 +132,70 @@ function recurse() {
-            // should be either "shared", "invited" or "joined"
-            // TODO: has doesn't seem to work
-            cy.get(`#historyVis-${data['historyVisibility']}`).parents("label").click();
-            cy.get(".mx_Dialog_cancelButton").click();
-            cy.get("[data-test-id=base-card-close-button]").click();
-            return "changed";
-        case "enable_dehydrated_device": {
-            cy.gotoAllSettings();
-            cy.get("[data-testid='settings-tab-USER_LABS_TAB']").click();
-            cy.get("[aria-label='Offline encrypted messaging using dehydrated devices']").click();
-            cy.get(".mx_Dialog_cancelButton").click();
-            runAction("enable_key_backup", data);
-            return "enabled_dehydrated_device";
-        }
-        case "enable_key_backup": {
-            cy.gotoAllSettings();
-            cy.get("[data-testid='settings-tab-USER_SECURITY_TAB']").click();
-            cy.get(".mx_SecureBackupPanel_buttonRow").contains("Set up").click();
-            cy.get(".mx_CreateSecretStorageDialog_optionIcon_securePhrase").click();
-            cy.get(".mx_CreateSecretStorageDialog [data-testid='dialog-primary-button']").click();
-            const password = data["key_backup_passphrase"];
-            if (!password) {
-                throw new Error("'key_backup_passphrase' not in data for action 'enable_dehydrated_device'");
-            }
-            cy.get(".mx_CreateSecretStorageDialog_passPhraseContainer input[type='password']").type(password);
-            cy.get("[data-testid='dialog-primary-button']").click();
-            // confirm the password again
-            cy.get(".mx_CreateSecretStorageDialog_passPhraseContainer input[type='password']").type(password);
-            cy.get("[data-testid='dialog-primary-button']").click();
-            // Continue to next screen
-            cy.get("[data-testid='dialog-primary-button']").click();
-            // Classic flakiness fix
-            cy.wait(500);
-            cy.get(".mx_CreateSecretStorageDialog").contains("Continue").click();
-            return "key_backup_enabled";
-        }
-        case "invite_user": {
-            cy.get(".mx_RightPanel_roomSummaryButton").click();
-            cy.get(".mx_RoomSummaryCard_icon_people").click();
-            cy.get(".mx_MemberList_invite").click();
-            cy.get(".mx_InviteDialog_addressBar input")
-                .type(`@${data["userId"]}`)
-                .type("{enter}");
-            cy.get(".mx_InviteDialog_goButton").click();
-            cy.get(".mx_AccessibleButton.mx_BaseCard_back").click();
-            cy.get(".mx_AccessibleButton.mx_BaseCard_close", { timeout: 30000 }).click();
-            return "invited";
-        }
-        case "open-room": {
-            cy.get(".mx_RoomSublist_tiles").contains(data["name"]).click();
-            return "room-opened";
-        }
+            return changeRoomHistoryVisibility(data);
+        case "open-room":
```

Comment:
```suggestion
        case "open-room":
        case "open_room":
```

Should we standardise on `this_case` as the format for action names - we can handle both at once as a temporary measure?

Reply:
yes, we should choose one and mention it in the trafficlight repo as a code style. My personal preference would be `open-room` but I'm okay with either. The proxy currently uses `camcelCase`, so I'll need to change that as well.